client/layout/sidebar/footer.jsx

```diff
@@ -15,8 +15,11 @@ import HappychatButton from 'components/happychat/button';
 const SidebarFooter = ( { translate, children } ) => (
 	<div className="sidebar__footer">
 		{ children }
-		<Button compact borderless href="/help">
-			<Gridicon icon="help-outline" /> { translate( 'Help' ) }
+		<Button borderless href={ config( 'signup_url' ) + '?ref=calypso-selector' }>
```

Comment:
Yes, I would say so. The benefit of this new design is that it can be the same everywhere. Help and Add New Site are both worth it, I would say, and the Chat button has the potential to become that.

Keep in mind that for launch, the Happychat button will *appear* when chat is available, or a chat hasn't ended. So it won't be a permanently visible button always.